media-fonts/fontawesome/fontawesome-4.7.0.recipe

```diff
@@ -0,0 +1,32 @@
+SUMMARY="A font containing scalable vector icons"
+DESCRIPTION="FontAwesome contains over 600 icons and is most widely used on \
+the internet to depict certain actions. Haiku's discussion forum at \
+https://discuss.haiku-os.org uses it as well.
+As long as Haiku doesn't support loading a font that's not installed, you \
+should install this webfont to see all the icons on the discussion forum."
+HOMEPAGE="http://fontawesome.io"
+COPYRIGHT="2013-2016 Dave Gandy"
+LICENSE="SIL Open Font License v1.1"
+REVISION="1"
+SOURCE_URI="https://github.com/FortAwesome/Font-Awesome/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="de512ba0e1dead382bbfce372cde74b3f18971d876fffb635ee9333f0db05d43"
+SOURCE_DIR=""
+DISABLE_SOURCE_PACKAGE=yes
```

Comment:
Suggestion (based on metrics :)
```diff
 SOURCE_DIR=""
-DISABLE_SOURCE_PACKAGE=yes
+
 ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE=yes
```

FWIW, we currently have 68 recipes with `DISABLE_SOURCE_PACKAGE` after `ARCHITECTURES`, and only 17 recipes with `DISABLE_SOURCE_PACKAGE` before `ARCHITECTURES`.